cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -639,5 +676,22 @@ struct value_accessor {
 
   __device__ T operator()(cudf::size_type i) const { return col.element<T>(i); }
 };
+
+template <typename T>
+struct mutable_value_accessor {
+  mutable_column_device_view col;  ///< mutable column view of column in device
+
+  /** -------------------------------------------------------------------------*
+   * @brief constructor
+   * @param[in] _col mutable column device view of cudf column
+   * -------------------------------------------------------------------------**/
+  mutable_value_accessor(mutable_column_device_view& _col) : col{_col} {
+    CUDF_EXPECTS(data_type(experimental::type_to_id<T>()) == col.type(),
+                 "the data type mismatch");
```

Comment:
Why is `CUDF_EXPECTS(!_col.has_nulls(), "Unexpected column with nulls.");` check skipped?

Reply:
It is up-to the user to update the `null_mask` as per the requirement.

Reply:
@karthikeyann